zitadel/terraform/main.tf

```diff
@@ -0,0 +1,95 @@
+terraform {
```

Comment:
the idea is that the same scripts must work for all environments. 
we need multiple varibles.tf files. We must hve one file called zitadel.testbed and one for zitadel.local
both wil have same values for now. except domain will be localhost, port 8080 and insecure true for zitadel.local

We must update document to show to how to use different varibles.tf

Reply:
fixed